e2e-tests/cypress/tests/integration/channels/enterprise/system_console/cluster_spec.js

```diff
@@ -39,21 +39,21 @@ describe('Cluster', () => {
             cy.get('.help-text').should('be.visible').and('have.text', contents);
 
             // * Verify that Experimental Gossip Encryption is set to false by default
-            cy.get('#EnableExperimentalGossipEncryptionfalse').should('have.attr', 'checked');
+            cy.get('#EnableGossipEncryptionfalse').should('have.attr', 'checked');
         });
 
         // # Enable Experimental Gossip Encryption
```

Comment:
I'm assuming this is part of what will go away completely? (this whole test?)

Reply:
Ah I didn't check deeply on the test. Let me run them and see what needs to be fixed. Thanks!